### PR TITLE
fix: tests use debug settings

### PR DESCRIPTION
**Pull request by erights**
I normally run with 
```sh
export LOCKDOWN_ERROR_TAMING=unsafe
```
This caused a failure in a test case comparing error messages, that were made more explanatory with this setting. The problem is that our tests were run with a neutral `lockdown` call that was letting this setting be set by the environment variable, but the test itself happened to depend on this option being `'safe'`. I think our best practice should be that `test-*.js` ava test files[1] should use the `@endo/init/debug.js` settings rather than the `@endo/init` settings. Then the error messages during testing will be more informative without endangering info leakage in production.

[1] aside from those whose purpose is to test the lockdown options, or whose purpose is to test that sensitive info in error messages get properly redacted.

### CODE REVIEW

File: packages/check-bundle/test/test-check-bundle.js
Comment (erights) [inline packages/check-bundle/test/test-check-bundle.js:79]: This was the case causing the failure.

**Comment by erights** [comment]
I left one `@endo/init` unchanged because it is in `packages/init` and so might be testing `init` itself. I'm not sure. That is `test-async-hooks.js`. I suspect this should be changed as well. Please let me know if I should change it as well.

**Comment by erights** [comment]
> I left one `@endo/init` unchanged because it is in `packages/init` and so might be testing `init` itself. I'm not sure. That is `test-async-hooks.js`. I suspect this should be changed as well. Please let me know if I should change it as well.

I have since changed my mind, and the PR no longer makes a special case for `test-async-hooks.js`. However, I still want feedback on this before merging.

File: packages/init/test/test-async_hooks.js
Comment (erights) [inline packages/init/test/test-async_hooks.js:5]: Reviewers, since this is in `packages/init`, I'd like feedback on whether this change is appropriate before merging.

Comment (kriskowal) [reply packages/init/test/test-async_hooks.js:5]: I defer to @mhofman’s opinion on this test.

**Review by kriskowal — APPROVED** [review]

File: packages/init/test/test-async_hooks.js
Comment (mhofman) [reply packages/init/test/test-async_hooks.js:5]: If debug transitively imports pre throught the package.json module map, i think it's ok

Comment (mhofman) [reply packages/init/test/test-async_hooks.js:5]: Thinking about it, it should since it's a public entry point itself.

**Review by mhofman — APPROVED** [review]

**Pull request merged**
